src/http/api/helpers/account.ts

```diff
@@ -0,0 +1,115 @@
+import { type Context, isActor } from '@fedify/fedify';
+import type { Account } from 'account/account.entity';
+import type { AccountService } from 'account/account.service';
+import type { Site } from 'account/types';
+import { getAccountHandle } from 'account/utils';
+import type { ContextData } from 'app';
+import {
+    getAttachments,
+    getFollowerCount,
+    getFollowingCount,
+    getHandle,
+    isFollowedByDefaultSiteAccount,
+} from 'helpers/activitypub/actor';
+import { sanitizeHtml } from 'helpers/html';
+import { lookupObject } from 'lookup-helpers';
+import type { AccountDTO } from '../types';
+
+/**
+ * Converts an Account to an AccountDTO
+ * @param account - The account to convert
```

Comment:
Nitpick: Space between the desc and param, no need for the ` - ` between the param and desc